client/me/purchases/index.js

```diff
@@ -138,47 +132,35 @@ export default ( router ) => {
 		clientRender
 	);
 
-	if ( config.isEnabled( 'purchases/new-payment-methods' ) ) {
-		router(
-			paths.addPaymentMethod( ':site', ':purchaseId' ),
-			sidebar,
-			siteSelection,
-			controller.changePaymentMethod,
-			makeLayout,
-			clientRender
-		);
-
-		router(
-			paths.changePaymentMethod( ':site', ':purchaseId', ':cardId' ),
-			sidebar,
-			siteSelection,
-			controller.changePaymentMethod,
-			makeLayout,
-			clientRender
-		);
-	} else {
-		router(
-			paths.addCardDetails( ':site', ':purchaseId' ),
-			sidebar,
-			siteSelection,
-			controller.addCardDetails,
-			makeLayout,
-			clientRender
-		);
+	router(
+		paths.addPaymentMethod( ':site', ':purchaseId' ),
+		sidebar,
+		siteSelection,
+		controller.changePaymentMethod,
+		makeLayout,
+		clientRender
+	);
 
-		router(
-			paths.editCardDetails( ':site', ':purchaseId', ':cardId' ),
-			sidebar,
-			siteSelection,
-			controller.editCardDetails,
-			makeLayout,
-			clientRender
-		);
-	}
+	router(
+		paths.changePaymentMethod( ':site', ':purchaseId', ':cardId' ),
+		sidebar,
+		siteSelection,
+		controller.changePaymentMethod,
+		makeLayout,
+		clientRender
+	);
 
 	// redirect legacy urls
 	router( '/me/billing', () => page.redirect( paths.billingHistory ) );
 	router( '/me/billing/:receiptId', ( { params: { receiptId } } ) =>
 		page.redirect( paths.billingHistoryReceipt( receiptId ) )
 	);
+	router( paths.addCardDetails( ':site', ':purchaseId' ), ( { params: { site, purchaseId } } ) =>
```

Comment:
🤔  Since it appears these routes were effectively removed by the flag until now, maybe they're not necessary and/or are covered by something else? Let's check.

Reply:
It's also possible that they are necessary, but were disabled accidentally as a result of our flag and we need to make sure that the legacy routes remain as you've done here; I just wanted to be sure that was the case.